### PR TITLE
docs: Typos and wording in integrating-with-linters.md

### PR DESCRIPTION
**Pull request by eduardopelitti**
This is just some minor grammar stuff I picked up as I was reading the docs. I think these changes makes the docs clearer.

Thanks for an awesome library!

## Description

Wording, typos and grammar changes for the Integrating with Linters section of the docs.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/integrating-with-linters.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ The downsides of those plugins are:
...
 
-Finally, we have tools that runs `prettier` and then immediately for example `eslint --fix` on files.
+Finally, we have tools that run `prettier` and then immediately apply `eslint --fix` on files.
 
 - [prettier-eslint](https://github.com/prettier/prettier-eslint)
 - [prettier-tslint](https://github.com/azz/prettier-tslint)
 - [prettier-stylelint](https://github.com/hugomrdias/prettier-stylelint)
 
-Those are useful if some aspect of Prettier’s output makes Prettier completely unusable to you. Then you can have for example `eslint --fix` fix that up for you. The downside is that these tools are much slower than just running Prettier.
+These are useful if some aspect of Prettier’s output makes Prettier completely unusable to you, then you can have `eslint --fix` fix that up for you. The downside is that these tools are much slower than just running Prettier.
```
Comment (thorn0) [inline docs/integrating-with-linters.md:40]: Removing "for example" isn't the right thing to do. Not all of these tools run ESLint.
Also, the comma makes the sentence strange, like it's a conditional sentence with two main clauses. "Then ..." is a separate thought here.

Comment (eduardopelitti) [reply docs/integrating-with-linters.md:40]: Yeah you're right, the comma does make it weird. It seems like an odd phrase to me, I'll try to work with it for a bit. And I'll restore the "for example" in the previous line as well.

Comment (thorn0) [reply docs/integrating-with-linters.md:40]: The general idea of those tools is that they run extra code transformations - namely linter autofixers - **after** Prettier.

**Comment by thorn0** [comment]
Closing for now, please feel free to reopen.

**Pull request closed without merging**